docs/Modern-RelayEnvironment.md

```diff
@@ -56,3 +56,37 @@ function handlerProvider(handle) {
   );
 }
 ```
+
+## Adding a `log` function
+
+You can get access to the inner life cycle stages of Relay, and log those. Super helpful for tracing, and debugging. For an in-depth look at what log events you can access, I urge you to try it out. Log a few things, and have a play. Alternively you can find a [list of possible events](https://github.com/facebook/relay/blob/37cee8039b888755e4ad4219e703be13a9a49ebb/packages/relay-runtime/store/RelayStoreTypes.js#L410) in the source code.
```

Comment:
I don't think the rest of the docs are in "I" form, so maybe drop that here too?